cmd/erasure.go

```diff
@@ -184,16 +186,16 @@ func getDisksInfo(disks []StorageAPI, endpoints []string) (disksInfo []madmin.Di
 			di := madmin.Disk{
 				Endpoint:       endpoints[index],
 				DrivePath:      info.MountPath,
-				TotalSpace:     info.Total,
-				UsedSpace:      info.Used,
-				AvailableSpace: info.Free,
+				TotalSpace:     xmath.Round(float64(info.Total)/humanize.GiByte, 1),
```

Comment:
Don't change the meaning of a field. This will make the numbers wrong between versions.